ruleguard/parser.go

```diff
@@ -21,9 +22,16 @@ import (
 
 // TODO(quasilyte): use source code byte slicing instead of SprintNode?
 
-type parseError string
+type parseError error
```

Comment:
I think this pattern is discouraged.
https://twitter.com/dvyukov/status/1174698980208513024
It can be a struct as ImportError, probably.

Reply:
ok, sure, I can make the change

Reply:
I made the change. I am marking resolved, let me know if this works for you.